lineage/figures/figureS12.py

```diff
@@ -0,0 +1,106 @@
+"""
+File: figureS12.py
+Purpose: Generates figure S12.
+Figure S12 shows the accuracy of state assignment versus 
+increasing cell numbers for 2-state, 3-state, and 4-state models.
+"""
+import numpy as np
+from ..Analyze import Analyze, Results
+from ..states.StateDistributionGaPhs import StateDistribution as StateDistPhase
+
+from .figureCommon import (
+    getSetup,
+    subplotLabel,
+    commonAnalyze,
+    figureMaker,
+    pi,
+    T,
+    E,
+    min_desired_num_cells,
+    lineage_good_to_analyze,
+    min_num_lineages,
+    max_num_lineages,
+    num_data_points,
+)
+from ..LineageTree import LineageTree
+
+
+def makeFigure():
+    """
+    Makes figure S12.
+    """
+
+    # Get list of axis objects
+    ax, f = getSetup((10, 10), (1, 3))
+
+    figureMaker(ax, *accuracy())
+
+    subplotLabel(ax)
+
+    return f
+
+
+def accuracy():
+    """
+    Calculates accuracy and parameter estimation
+    over an increasing number of lineages in a population for
+    a uncensored two-state model.
+    We increase the desired number of cells in a lineage by
+    the experiment time.
+    """
+
+    pi2 = np.array([0.60, 0.40])
+    T2 = np.array([[0.75, 0.25], [0.1, 0.90]])
+    E2 = [StateDistPhase(0.99, 0.9, 20, 5, 10, 3), StateDistPhase(0.88, 0.75, 10, 2, 15, 4)]
+    
+    X_2 = [LineageTree.init_from_parameters(pi2, T2, E2, desired_num_cells=(2 ** 11) - 1)]
```

Comment:
This line, and similarly, X_3 and X_4 that you have here, will create only one lineage. Ideally, we would want to analyze a population with more than 1 lineage. You can look at these lines in figure4.py and do a slight change to create a population with increasing cell number *and* lineage number.
`num_lineages = np.linspace(min_num_lineages, int(0.35 * max_num_lineages), num_data_points, dtype=int)
    experiment_times = np.linspace(1000, int(2.5 * 1000), num_data_points)
    list_of_populations = []
    list_of_fpi = []
    list_of_fT = []
    list_of_fE = []
    for indx, num in enumerate(num_lineages):
        population = []
        for _ in range(num):
            good2go = False
            while not good2go:
                tmp_lineage = LineageTree.init_from_parameters(pi, T, E2, desired_num_cells=min_desired_num_cells, censor_condition=3, desired_experiment_time=experiment_times[indx])
                good2go = lineage_good_to_analyze(tmp_lineage)
            population.append(tmp_lineage)
        # Adding populations into a holder for analysing
        list_of_populations.append(population)`